src/ToricVarieties.jl

```diff
@@ -115,52 +115,113 @@ function projective_space( d::Int )
     pmntv = Polymake.fulton.NormalToricVariety(f)
     
     # wrap it and return
-    return normalToricVariety( variety, pmntv )
+    return NormalToricVariety( variety, pmntv )
 end
 export projective_space
 
 
+"""
+    hirzebruch_surface( r::Int )
+
+Constructs the r-th Hirzebruch surface.
+
+# Examples
+```julia-repl
+julia> hirzebruch_surface( 5 )
+NormalToricVariety(GAP: <A toric variety of dimension 2>, Polymake.BigObjectAllocated(Ptr{Nothing} @0x0000562fbcc21b70))
+```
+"""
+function hirzebruch_surface( r::Int )
+    Rays = [ 1 0; 0 1; -1 r; 0 -1]
+    Cones = [[1,2],[2,3],[3,4],[4,1]]
+    return NormalToricVariety( Rays, Cones )
+end
+export hirzebruch_surface
+
+
+"""
+    delPezzo( b::Int )
+
+Constructs the delPezzo surface with b blowups for b at most 3.
+
+# Examples
+```julia-repl
+julia> del_pezzo( 3 )
+NormalToricVariety(GAP: <A toric variety of dimension 2>, Polymake.BigObjectAllocated(Ptr{Nothing} @0x0000562fbcc21b70))
+```
+"""
+function del_pezzo( b::Int )
+    if b < 0
+        @warn("Number of blowups for construction of delPezzo surfaces must be non-negative.")
+        return 0
+    end
+    if b == 0 
+        return projective_space( 2 )
+    end
+    if b == 1
+        Rays = [ 1 0; 0 1; -1 0; -1 -1 ]
+        Cones = [ [1,2],[2,3],[3,4],[4,1] ]
+        return NormalToricVariety( Rays, Cones )
+    end
+    if b == 2
+        Rays = [ 1 0; 0 1; -1 0; -1 -1; 0 -1 ]
+        Cones = [ [1,2],[2,3],[3,4],[4,5],[5,1] ]
+        return NormalToricVariety( Rays, Cones )
+    end
+    if b == 3
+        Rays = [ 1 0; 1 1; 0 1; -1 0; -1 -1; 0 -1 ]
+        Cones = [ [1,2],[2,3],[3,4],[4,5],[5,6],[6,1] ]
+        return NormalToricVariety( Rays, Cones )
+    end
+    if b > 3
+        @warn("delPezzo surfaces with more than 3 blowups are realized as subvarieties of toric ambient spaces. This is currently not supported.")
+        return 0
+    end
+end
+export del_pezzo
+
+
 ######################
 # 4: Properties
 ######################
 
 
 """
-    is_normal_variety( v::normalToricVariety )
+    is_normal_variety( v::NormalToricVariety )
 
 Checks if the normal toric variety `v` is normal. (This function is somewhat tautological at this point.)
 
 # Examples
 ```julia-repl
-julia> is_normal_variety( projective_space(  2 ) )
+julia> is_normal_variety(projective_space( 2 ))
 true
 ```
 """
-function is_normal_variety( v::normalToricVariety )
+function is_normal_variety( v::NormalToricVariety )
     return true
 end
 export is_normal_variety
 
 
 """
-    is_affine( v::normalToricVariety )
+    is_affine( v::NormalToricVariety )
 
 Checks if the normal toric variety `v` is affine.
 
 # Examples
 ```julia-repl
-julia> is_normal_variety( projective_space(  2 ) )
+julia> is_normal_variety( projective_space( 2 ) )
 false
 ```
 """
-function is_affine( v::normalToricVariety )
+function is_affine( v::NormalToricVariety )
     return v.polymakeNTV.AFFINE
```

Comment:
Is this a always a boolean? Perhaps help the compiler via an annotation:
```suggestion
    return v.polymakeNTV.AFFINE::Bool
```